man/ocitools-validate.1.md

```diff
@@ -18,6 +21,9 @@ Validate an OCI bundle
 **--path=PATH
    Path to bundle
 
+**--hooks-check**
```

Comment:
Maybe there will be other particular checks, but I still think `--hooks-check` is better and other particular checks in the future should be with `-check`.  it obviously tell people what it is used for.